src/modules/add-comment.js

```diff
@@ -0,0 +1,26 @@
+async function addComment(itemid, usrName, usrComment) {
```

Comment:
- When we look at the learning objectives of this capstone project, we find this objective 👇🏼:

  > - Use ES6 syntax.
  
  Therefore, we are also looking for strict adherence to ES6 syntax when implementing our functions in this review.
  
  Please refactor all functions that are not in ES6 syntax to be ES6 compliant. We'll need to remember about `arrow functions` here:
  
  ```JavaScript
  const functionName = (params_if_any) => {
    // function body here
  }
  ```
  
  If you would like to read more about arrow functions, check [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) out.